docs/reference/filters.md

```diff
@@ -466,6 +466,23 @@ Example:
 * -> randomContent(42) -> <shunt>;
 ```
 
+## backendTimeout
+
+Configure backend timeout. Skipper responds with `504 Gateway Timeout` status if obtaining a connection,
+sending the request, and reading the backend response headers and body takes longer than the configured timeout.
+However, if response streaming has already started it will be terminated, i.e. client will receive backend response
+status and truncated response body.
```

Comment:
I think we should include here that if the request body is received too slowly, that also cancels the flow.

Reply:
Doesn't `sending the request` covers it?

Reply:
true, sgtm